node_modules/gh-users/lib/rest.js

```diff
@@ -82,7 +84,13 @@ GrassHopper.appRouter.on('get', '/api/users/:id', getUser);
  * @Return      {Me}                                        The current user
  */
 GrassHopper.appRouter.on('get', '/api/me', function(req, res) {
-    res.status(200).send(req.ctx.user);
+    var me = {
```

Comment:
Not yet I think. This stuff is only useful in the REST layer. I don't think we need to add this in our API layer just yet